Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/UnsubscribeTransforms.java

```diff
@@ -56,7 +56,6 @@ public LogEventRequestType transformNhinUnsubscribeRequestToAuditMessage(LogNhin
         AuditMessageType auditMsg = new AuditMessageType();
```

Comment:
This class can be removed. These are part of HIEM and that service implementation is removed from codebase.